### PR TITLE
Bump jcasc version to post security fixes

### PR DESCRIPTION
**Pull request by timja**
see:
https://github.com/jenkinsci/ssh-slaves-plugin/pull/147
https://github.com/jenkinsci/bom/pull/78

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.138.4</jenkins.version>
     <java.level>8</java.level>
-    <jcasc.version>1.20</jcasc.version>
+    <jcasc.version>1.30-20190908.202458-1</jcasc.version>
```
Comment (casz) [inline pom.xml]:
```suggestion
    <jcasc.version>1.30</jcasc.version>
```

**Comment by casz** [comment]
@jvz would appreciate a release with this fix as it helps resolve https://github.com/jenkinsci/bom/pull/78

**Comment by casz** [comment]
Jenkins ci seems foobar!

**Comment by timja** [comment]
could we get a release please @jvz? it would unblock: https://github.com/jenkinsci/bom/pull/78

thanks

**Comment by jvz** [comment]
Sure, I'll make one today. Thanks for the reminder.

**Comment by jvz** [comment]
Released in 1.17.2

**Comment by casz** [comment]
Thanks ❤️

**Pull request merged**